liana-gui/src/app/view/receive.rs

```diff
@@ -33,103 +34,220 @@ use crate::{
+                                        scrollable(
+                                            Column::new()
+                                                .push(Space::with_height(Length::Fixed(10.0)))
+                                                .push(
+                                                    text(
+                                                        prev_labels
+                                                            .get(&address.to_string())
+                                                            .cloned()
+                                                            .unwrap_or_default(),
+                                                    )
+                                                    .small()
+                                                    .style(theme::text::secondary),
                                                 )
-                                                .on_press(Message::Select(i)),
-                                            )
-                                            .push(Space::with_width(Length::Fill))
-                                            .push(
-                                                button::secondary(None, "Show QR Code")
-                                                    .on_press(Message::ShowQrCode(i)),
+                                                // Space between the label and the scrollbar
+                                                .push(Space::with_height(Length::Fixed(10.0))),
+                                        )
+                                        .direction(
+                                            scrollable::Direction::Horizontal(
+                                                scrollable::Scrollbar::new()
+                                                    .width(2)
+                                                    .scroller_width(2),
                                             ),
+                                        ),
                                     )
-                                    .spacing(10),
-                            )
-                            .padding(20),
+                                    .padding(10)
+                                    .width(Length::FillPortion(7)),
+                                )
+                                .align_y(Alignment::Center),
                         )
-                    },
-                )),
+                        .on_press(Message::SelectAddress(address.clone()))
+                        .style(theme::button::secondary)
+                    } else {
+                        // Continue the row index from those of generated addresses above.
+                        Button::new(address_card(
+                            addresses_count + i,
+                            address,
+                            prev_labels,
+                            labels_editing,
+                        ))
```

Comment:
```suggestion
                        ))
                        .padding(0)
```
Remove the default padding, otherwise the green border does not match the card border.

Reply:
Done.